src/config/configAggregator.ts

```diff
@@ -78,14 +78,14 @@ export interface ConfigInfo {
  * ```
  */
 export class ConfigAggregator extends AsyncOptionalCreatable<JsonMap> {
-  private static instance: AsyncOptionalCreatable;
+  private static instance: AsyncOptionalCreatable | undefined;
```

Comment:
Why did this need to change?

Reply:
@mdonnalley I will look, but I recall hitting a test failure where a call is being made directly to buildJwt... function, without calling create or new on a ConfigAggregator.